storybook/src/server/framework-preset.ts

```diff
@@ -17,36 +14,39 @@ import dedent from 'ts-dedent';
 type BuildOptions = {
     twigComponent: TwigComponentConfiguration;
     twig: TwigConfiguration;
-    runtimeDir: string;
-    projectDir: string;
     additionalWatchPaths: string[];
+    templatePathAliases: {
+        [p: string]: string;
+    };
 };
 
 const getBuildOptions = async (symfonyOptions: SymfonyOptions) => {
-    const projectDir = await getKernelProjectDir();
-    const twigComponentsConfig = await getTwigComponentConfiguration();
-    const twigConfig = await getTwigConfiguration();
+    const symfonyConfig = await getSymfonyConfig(symfonyOptions.storybookCachePath);
 
     const componentNamespaces: { [p: string]: string[] } = {};
 
-    const twigPaths: string[] = Object.keys(twigConfig.paths).map((key) => `${projectDir}/${key}/`);
+    const twigPaths: string[] = Object.keys(symfonyConfig.twig_config.paths).map((key) => {
+        if (key.startsWith(symfonyOptions.projectDir)) {
+            return `${key}/`;
+        }
+
+        return `${symfonyOptions.projectDir}/${key}/`;
+    });
 
     if (twigPaths.length === 0) {
-        twigPaths.push(`${projectDir}/templates`);
+        twigPaths.push(`${symfonyOptions.projectDir}/templates`);
     }
 
     for (const { name_prefix: namePrefix, template_directory: templateDirectory } of Object.values(
-        twigComponentsConfig.defaults
+        symfonyConfig.twig_component_config.defaults
     )) {
         componentNamespaces[namePrefix] = twigPaths.map((twigPath) => join(twigPath, templateDirectory));
     }
 
     const anonymousNamespace: string[] = twigPaths.map((twigPath) =>
-        join(twigPath, twigComponentsConfig['anonymous_template_directory'])
+        join(twigPath, symfonyConfig.twig_component_config.anonymous_template_directory)
     );
 
-    const runtimeDir = (await getBundleConfig()).runtime_dir;
```

Comment:
Not sure about the usage of `runtime_dir`. I didn't find anything in source code